internal/cmd/commands/ferry/pause.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package ferry
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*PauseCommand)(nil)
+	_ cli.CommandAutocomplete = (*PauseCommand)(nil)
+)
+
+type PauseCommand struct {
+	*base.Command
+}
+
+func (c *PauseCommand) Synopsis() string {
+	return "Pauses the running boundary ferry daemon"
+}
+
+func (c *PauseCommand) Help() string {
+	helpText := `
+Usage: boundary ferry pause
+
+  Pause the boundary ferry daemon:
+
+      $ boundary ferry pause
+
+  For a full list of examples, please see the documentation.
+	
+` + c.Flags().Help()
+	return strings.TrimSpace(helpText)
+}
+
+func (c *PauseCommand) Flags() *base.FlagSets {
+	set := c.FlagSet(base.FlagSetOutputFormat)
+	f := set.NewFlagSet("Client Options")
+
+	f.BoolVar(&base.BoolVar{
+		Name:   "output-curl-string",
+		Target: &c.FlagOutputCurlString,
+		Usage:  "Instead of executing the request, print an equivalent cURL command string and exit.",
+	})
+
+	f.UintVar(&base.UintVar{
+		Name:    "ferry-port",
+		Target:  &c.FlagFerryDaemonPort,
+		Default: 9300,
+		EnvVar:  base.EnvFerryDaemonPort,
+		Usage:   "The port on which the ferry daemon is listening.",
+	})
+
+	return set
+}
+
+func (c *PauseCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictNothing
+}
+
+func (c *PauseCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *PauseCommand) Run(args []string) int {
+	ctx := c.Context
+	f := c.Flags()
+	if err := f.Parse(args); err != nil {
+		c.PrintCliError(err)
+		return base.CommandUserError
+	}
+
+	resp, apiErr, err := c.Pause(ctx)
+	if err != nil {
+		c.PrintCliError(err)
+		return base.CommandCliError
+	}
+	if apiErr != nil {
+		c.PrintApiError(apiErr, "Error from ferry daemon when attempting to pause")
+		return base.CommandApiError
+	}
+
+	if resp.StatusCode() != 200 {
+		return base.CommandCliError
+	}
+	return base.CommandSuccess
+}
+
+func (c *PauseCommand) Pause(ctx context.Context) (*api.Response, *api.Error, error) {
+	const op = "ferry.(PauseCommand).Pause"
+	client := retryablehttp.NewClient()
+	client.Logger = nil
+	client.RetryWaitMin = 100 * time.Millisecond
+	client.RetryWaitMax = 1000 * time.Millisecond
```

Comment:
This shouldn't require a retryable client, Ferry doesn't have rate limits. If this is the convention in these files then fine but just noting it.